dominio/SolicitacaoAcaoDTO.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dominio
+{
+    public class SolicitacaoAcaoDTO
+    {
+        public string Escola { get; set; }
+        public string NomeSolicitante { get; set; }
+        public string VinculoEscola { get; set; }
+        public string Email { get; set; }
+        public string Telefone { get; set; }
+        public string CiclosEnsino { get; set; }
```

Comment:
Como "Ciclos de ensino" é um campo de seleção múltipla no front, aqui é necessário que seja um array de string, e não apenas uma string.
```suggestion
        public string[] CiclosEnsino { get; set; }
```